reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Migrating from Windows PowerShell 5.1 to PowerShell 7
+description: Update from PowerShell 5.1 to PowerShell 7 for your Windows platforms.
+ms.date: 03/25/2020
+---
+
+# Migrating from Windows PowerShell 5.1 to PowerShell 7
+
+Designed for cloud, on-premises, and hybrid environments, PowerShell 7 is packed with
+enhancements and [new features](What-s-New-in-PowerShell-70.md).
+
+- Installs and runs side-by-side with Windows PowerShell
+- Improved compatibility with existing Windows PowerShell modules
+- New language features, like ternary operators and `ForEach-Object -Parallel`
+- Improved performance
+- SSH-based remoting
+- Cross-platform interoperability
+- Support for Docker containers
+
+PowerShell 7 works side-by-side with Windows PowerShell letting you easily test and compare before
+deployment. Migration is simple, requiring a minimal investment of time and lower the risk of
+failure.
+
+PowerShell 7 is supported on the following Windows operating systems:
+
+- Windows 8.1 and 10
+- Windows Server 2012, 2012 R2, 2016, and 2019
+
+PowerShell 7 also runs on macOS and several Linux distributions. For a list of supported operating
+systems and information about the support lifecycle, see the
+[PowerShell Support Lifecycle](/powershell/scripting/powershell-support-lifecycle).
+
+## Installing PowerShell 7
+
+For flexibility and to support the needs of IT, DevOps engineers, and developers, there are several
+options available to install PowerShell 7. In most cases, the installation options can be reduced to
+the following methods:
+
+- Deploy PowerShell using the [MSI package](/powershell/scripting/install/installing-powershell-core-on-windows#installing-the-msi-package)
```

Comment:
May be worth calling out that the MSI requires admin whereas the zip does not.

Reply:
Done